src/test/java/create_task_use_case/CreateTaskDataAccessTest.java

```diff
@@ -0,0 +1,31 @@
+package create_task_use_case;
+
+import data_access.DataAccessDay;
+import data_access.DataAccessTask;
+import org.junit.jupiter.api.Test;
+
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CreateTaskDataAccessTest {
```

Comment:
Probably test the save() method of your DataAccess class here as well.

Reply:
Added.